packages/x-date-pickers/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig",
   "compilerOptions": {
-    "types": ["react", "mocha", "node"]
+    "types": ["react", "mocha", "node"],
+    "noImplicitAny": false
```

Comment:
@mui/explore You might want to enforce this flag in the pickers packages (currently TS emits errors when the flag is enabled)

Reply:
Thanks for noting 👍